src/resources/listings.js

```diff
@@ -99,6 +108,63 @@ class Listings extends ResourceBase {
     return range(0, Number(listingsLength))
   }
 
+  async search(query) {
+    // Issues an API call to the bridge server for searching listings.
+    // Args:
+    //   query(SearchQuery): query object.
+    // Returns:
+    //   list(int): listing ids that match the query.
+    if (query.rawQuery == '') {
```

Comment:
It's generally considered best practice in javascript to use the strict equality operator `===`. See https://stackoverflow.com/a/359509